src/test/java/org/opensrp/repository/postgres/ProductCatalogueRepositoryTest.java

```diff
@@ -84,7 +84,8 @@ public void testGetById() {
 	public void testGetProductCataloguesBySearchBean(){
 		ProductCatalogueSearchBean productCatalogueSearchBean = new ProductCatalogueSearchBean();
 		productCatalogueSearchBean.setUniqueId(1l);
-		List<ProductCatalogue> productCatalogues = productCatalogueRepository.getProductCataloguesBySearchBean(productCatalogueSearchBean, "http://localhost:8080/opensrp");
+		int noLimit = Integer.MAX_VALUE;
+		List<ProductCatalogue> productCatalogues = productCatalogueRepository.getProductCataloguesBySearchBean(productCatalogueSearchBean, noLimit, "http://localhost:8080/opensrp");
```

Comment:
I think we should add new test cases for this and still leave the unit test for the deprecated methods in place (until they are removed)